File: uri/soma/soma.pas
program foo(Input, Output);

function soma( a, b : Integer) : Integer;
begin
    soma := a + b;
end;

var 
    a, b, tot: Integer;

begin
    readln(a);
    readln(b);
    tot := soma(a, b);
    writeln('SOMA = ', tot);
end.

File: uri/soma/soma.cpp
#include <iostream>

using namespace std;

class Soma{

    public:

    int soma(int a, int b)
    {
        return a + b;
    }

};

int main(){

    int a, b;
    Soma foo;
    cin >> a;
    cin >> b;    
    cout << "SOMA = " << foo.soma(a, b) << "\n";
    return 0;
}